src/test/java/tests/LoginTests.java

```diff
@@ -33,4 +34,13 @@ public void tc02_LoginWithBlankUsernameTextbox() {
         Assert.assertEquals(loginPage.getErrorEmailMessage(), "You must specify a username.", "Error email message doesn't display correctly");
         Assert.assertEquals(loginPage.getErrorLoginMessage(), "There was a problem with your login and/or errors exist in your form.", "Error login message doesn't display correctly");
     }
+
+    @Test(description = "User cannot log into Railway with invalid password")
+    public void tc03_LoginWithInvalidPassword() {
+        LogHelper.info("Enter invalid password");
```

Comment:
log step is not correct, method is working for "Login with invalid password" not just "Enter invalid password", please update